src/cache.cpp

```diff
@@ -132,14 +132,15 @@ namespace {
 		return (cache[key] = {bmp, Game_Clock::GetFrameTime()}).bitmap;
 	}
 
-	BitmapRef LoadBitmap(const std::string& folder_name, const std::string& filename,
+	BitmapRef LoadBitmap(StringView folder_name, StringView filename,
 						 bool transparent, const uint32_t flags) {
 		const auto key = MakeHashKey(folder_name, filename, transparent);
 
 		auto it = cache.find(key);
 
 		if (it == cache.end()) {
-			const std::string path = FileFinder::FindImage(folder_name, filename);
+			// FIXME: STRING_VIEW string copies here
```

Comment:
Is this fixme for later PRs?

Reply:
Yeah, I didn't fully StringView'ify filefinder.

Reply:
I would prefer it when you don't touch the FileFinder too much otherwise my next big FS PR will be terrible to rebase.